src/utilities/getTokenIdFromVAddress.ts

```diff
@@ -0,0 +1,14 @@
+import vBepTokens from 'constants/contracts/addresses/vBepTokens.json';
+import { TokenId } from 'types';
+
+const getTokenFromVAddress = (address: string) => {
+  const token = Object.entries<{ [key: string]: string }>(vBepTokens).find(
+    ([, value]) =>
+      value[process.env.REACT_APP_CHAIN_ID || '56'].toLowerCase() === address.toLowerCase(),
```

Comment:
We should use the `isOnTestnet` config variable instead.

Reply:
I don't think that makes sense here. The unary is here because  `process.env.REACT_APP_CHAIN_ID` could be undefined

Reply:
Ah right my bad, I read this wrong! You're right